sdk/lib/dependencies/dependencies.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  Effects,
+  PackageId,
+  DependencyRequirement,
+  SetHealth,
+  CheckDependencyResult,
+} from "../types"
+
+export type CheckAllDependencies = {
+  notRunning: () => Promise<CheckDependencyResult[]>
+
+  notInstalled: () => Promise<CheckDependencyResult[]>
+
+  errorMessages: () => Promise<{ [id: string]: SetHealth[] }>
```

Comment:
Why SetHealth?

Reply:
Because that is the return type of the result, I'm just passing through the types
![image](https://github.com/Start9Labs/start-os/assets/2364004/4c848bb1-1e04-47af-8b49-d92c3175f151)
https://github.com/Start9Labs/start-os/pull/2609/files/9b94b8035039c436ebee4db1a0fdb71e552564ef#diff-448f8e9b112af283ba61fd29e758ef1a0c6f33206f3acd03fff0b9e7fb9f7a80R603

Reply:
It seems the field is improperly named then, if it is specific to health checks